### DISCUSSION TITLE
Unpause personal account

### DISCUSSION THREAD

**Discussion by NirachaMarchett** [vercel/community#4183] [Account Issue] [answered]
### Question

Hi everyone,

Right now, my hobby account is being paused and all the project in this account already has been removed for a while now. Is it possible to unpause my personal account? What would happen if I delete my personal account, will I be able to sign up for hobby account again?

I also have the pro account, if i delete personal account, will it delete my pro account?

Thank you so much in advance.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps.

For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)

To open a ticket please use the appropriate channel found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
